resolwe/storage/connectors/transfer.py

```diff
@@ -189,11 +215,15 @@ class Transfer:
     """Transfer data between two storage connectors using in-memory buffer."""
 
     def __init__(
-        self, from_connector: BaseStorageConnector, to_connector: BaseStorageConnector
+        self,
+        from_connector: BaseStorageConnector,
+        to_connector: BaseStorageConnector,
+        retry_count: int = 3,
     ):
         """Initialize transfer object."""
         self.from_connector = from_connector
         self.to_connector = to_connector
+        self._retry_count = retry_count
```

Comment:
This variable looks obsolete

Reply:
True. I have removed it from the source.